app/controllers/api/v1/registrant/base_controller.rb

```diff
@@ -22,6 +22,15 @@ def bearer_token
           header.gsub(pattern, '') if header&.match(pattern)
         end
 
+        def associated_domains(user)
+          country_code, ident = user.registrant_ident.split('-')
```

Comment:
This part is crying about being encapsulated into `RegistrantUser` :)

Reply:
You're correct, there is even another place where we use a similar block:
https://github.com/internetee/registry/blob/master/app/controllers/registrant/domains_controller.rb#L56-L65

But I'd do it in a separate branch to avoid broad surface area for the manual tests, after this one is in master.

Reply:
I saw some additional chunks that use that `ident` of a `RegistrantUser`.